vignettes/methodological_introduction.Rmd

```diff
@@ -0,0 +1,109 @@
+---
+title: "What are MMRMs? A brief introduction"
+package: mmrm
+bibliography: '`r system.file("REFERENCES.bib", package = "mmrm")`'
+output:
+  rmarkdown::html_document:
+          theme: "spacelab"
+          highlight: "kate"
+          toc: true
+          toc_float: true
+vignette: |
+  %\VignetteIndexEntry{What are MMRMs? A brief introduction}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+editor_options:
+  chunk_output_type: console
+  markdown:
+    wrap: 72
+---
+
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+
+## Abstract
+
+Mixed models for repeated measures (MMRMs) are frequently used in the analysis of data from clinical trials. 
+They are specifically suited to model continuous variables that were repeatedly measured at discrete time points (or within defined time-windows).
+In a clinical trial, these time points are typically visits according to a schedule that is pre-defined in the trial protocol.
+The distinguishing feature of MMRMs, compared to other implementations of linear mixed models, is that subject-specific random effects (which are not of direct interest for estimation and inference) are considered as residual effects, i.e. they are part of the error correlation matrix.
+This vignette provides a brief methodological introduction to MMRMs.
+MMRMs are described as an extension to a basic linear mixed-effects model.
+The aim is to provide a basic orientation and guide for applied statisticians working in the field of clinical trials regarding theoretical underpinnings of MMRMs. In the descriptions of the models below, we generally follow @Pinheiro2000 and @Mallinckrodt2017. 
+
+
+## The basic linear mixed-effects model
+
+@Laird1982 introduced the basic linear mixed-effects model for a single level of grouping to be any model that expresses an $n_i$-dimensional response (column) vector $y_i$ for the $i$th subject (or, more generally, group or unit) as
+
+\[
+ y_i = X_i \beta + Z_i b_i + \epsilon_i, \quad i=1,\ldots,M ,\\
+ b_i \sim \mathcal{N}(0,\Psi), \quad \epsilon_i \sim \mathcal{N}(0,\sigma^2I),
+\]
+where 
+$\beta$ is the $p$-dimensional vector of fixed effects,
+$b$ is the $q$-dimensional vector of random patient-specific effects, 
+$X_i$ (of size $n_i \times p$) and $Z_i$ (of size $n_i \times q$) are known regressor matrices relating observations to the fixed-effects and random-effects, respectively, and 
+$\epsilon_i$ is the $n_i$-dimensional within-subject error.
```

Comment:
maybe could be nice with bullet points?

Reply:
Yes. I now use bullet points, which improves readability.